core/model/modx/modx.class.php

```diff
@@ -1548,7 +1550,8 @@ public function regClientStartupScript($src, $plaintext= false) {
             } elseif (strpos(strtolower($src), "<script") !== false) {
                 $this->sjscripts[count($this->sjscripts)]= $src;
             } else {
-                $this->sjscripts[count($this->sjscripts)]= '<script src="' . $src . '"></script>';
+                $scriptOptions = implode(' ', $options);
+                $this->sjscripts[count($this->sjscripts)]= "<script $scriptOptions src=\"$src\"></script>";
```

Comment:
If `$options` is empty then this will add a superfluous space. What do you think about pushing the src attribute and it's value to the end of the `$options` array?

Reply:
OK, good idea